util/devel/test/portability/provision-scripts/chapel-default.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 
+PROVISION_SCRIPTS_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
 if [ ! -d chapel ]
 then
-  ../../provision-scripts/chapel-update.sh
+  $PROVISION_SCRIPTS_DIR/chapel-update.sh
 fi
 
-source ../../provision-scripts/chapel-setmakej.sh
+source $PROVISION_SCRIPTS_DIR/chapel-setmakej.sh
 
-echo "cd chapel && source util/setchplenv.bash && make $MAKEJ && make check"
-cd chapel && source util/setchplenv.bash && make $MAKEJ && make check
+echo "cd chapel && source util/setchplenv.bash && make $MAKEJ && make $MAKEJ check"
+cd chapel && source util/setchplenv.bash && make $MAKEJ && make $MAKEJ check
```

Comment:
this assumes the script is being run from effectively `$CHPL_HOME/..`, which seems counter-intuitive. Maybe consider making this based on `$PROVISION_SCRIPTS_DIR`.

Reply:
unless the idea is that you are sparse-cloning these scripts and then using an external tarball?